run.py

```diff
@@ -392,6 +392,10 @@ def setup(self):
     def check_skip(self):
         if not "skip" in self.config:
             return
+        if isinstance(self.config["skip"], bool):
+            if self.config["skip"]:
+                raise UnsatisfiedRequirementError("skipped by default")
+            return
```

Comment:
nit: This return statement can be skipped too since we're using a `raise` in the statement above this.

Reply:
Without a return here the next line will try to iterate on a boolean.

Reply:
Correct. I did not test the False case at all. Sorry about this as well.